blueprint-manager/src/test/java/com/sequenceiq/cloudbreak/blueprint/kerberos/KerberosDetailServiceTest.java

```diff
@@ -12,24 +13,89 @@
 @RunWith(MockitoJUnitRunner.class)
 public class KerberosDetailServiceTest {
 
-    private KerberosDetailService underTest = new KerberosDetailService();
+    private static final String TEST_DEFAULT_HOST = "some default host value";
```

Comment:
The test data doesn't look like a default host

Reply:
it shouldn't be, because the test does not verify the content so for test purpose this value could be anything